docs/developer/products.mdx

```diff
@@ -509,14 +513,94 @@ Response:
 }
 ```
 
-- `"quantityAvailable": 50`: customers only see maximum of 50 pcs available to buy
-- `stocks` contain full list of stock quantities in every of the warehouses
+- The `quantityAvailable` is a sum of stock quantities from all available stocks.
+  In this case, the sum is bigger than the `MAX_CHECKOUT_LINE_QUANTITY` so the maximum
+  value of 50 products is returned.
+- `stocks` contain a full list of stock quantities in each warehouse.
+
+:::note
+When the product has a track inventory turned off, the `quantityAvailable` will be always
+equal to the `MAX_CHECKOUT_LINE_QUANTITY` value.
+:::
+
+### Getting the available quantity for the particular address
+
+The quantity available can be fetched for the given address. The returned quantity
+is the sum of the quantity from all stocks that are available in the shipping zone that
+supports provided address. When the address is not provided the highest quantity
+from available shipping zones is returned.
```

Comment:
```suggestion
supports provided address. When the address is not provided, the highest quantity
from available shipping zones is returned.
```

Reply:
What does "available" shipping zones mean? Does it mean "from all available shipping zones in the system"?

Reply:
IMO, the `available shipping zones` are explained in the previous sentence

> The returned quantity is the sum of the quantity from all stocks that are available in the shipping zone that supports provided address.